lib/openstudio-standards/prototypes/common/objects/Prototype.hvac_systems.rb

```diff
@@ -6475,4 +6475,227 @@ def model_add_hvac_system(model,
     rename_air_loop_nodes(model)
     rename_plant_loop_nodes(model)
   end
+
+  # This method will add an swh water fixture to the model for the space.
+  # if the it will return a water fixture object, or NIL if there is no water load at all.
+  def model_add_swh_end_uses_by_spaceonly(model, space, swh_loop)
```

Comment:
this should go in the service water heating section

Reply:
I deleted this section; it is overloaded in the NECB code and only used there, so it doesn't need to be in hvac_systems.rb